src/server/models/Recipes.js

```diff
@@ -18,3 +18,5 @@ let RecipeSchema = new mongoose.Schema(
 mongoose.model('Recipe', RecipeSchema);
```

Comment:
Are lines 18-20 necessary? This is is app.js

Reply:
Not sure, wasn't my idea

Reply:
We can try to refactor if we have more time later